jnigen/example/in_app_java/lib/android_utils.dart

```diff
@@ -46,8 +46,12 @@ class AndroidUtils extends jni.JObject {
 
   /// from: static public void showToast(android.app.Activity mainActivity, java.lang.CharSequence text, int duration)
   static void showToast(
-          jni.JObject mainActivity, jni.JObject text, int duration) =>
-      _showToast(mainActivity.reference, text.reference, duration).check();
+    jni.JObject mainActivity,
+    jni.JObject text,
+    int duration,
```

Comment:
Can we land this last comma format change separately? It would make it easier to review.

(Sidenode, I indeed prefer having a trailing comma, it uses more lines but makes code easier to read in general.)

Reply:
The reason I added this is because there may or may not be a `{...}` for the named (type) parameters. I could add some `if` statement to only add the comma if we have type parameters.